bubbletea/tea_other.go

```diff
@@ -5,20 +5,27 @@ package bubbletea
 
 import (
 	tea "github.com/charmbracelet/bubbletea"
-	"github.com/charmbracelet/lipgloss"
 	"github.com/charmbracelet/ssh"
-	"github.com/muesli/termenv"
 )
 
 func makeOpts(s ssh.Session) []tea.ProgramOption {
+	pty, _, ok := s.Pty()
+	environ := s.Environ()
+	if pty.Term != "" {
+		environ = append(environ, "TERM="+pty.Term)
```

Comment:
isn't TERM already in s.Environ?

Reply:
I don't think so, `TERM` is part of the SSH `pty-req` while environ variables are part of the `env` SSH request.
See https://www.rfc-editor.org/rfc/rfc4254#section-6.2

Reply:
`TERM` shouldn't be exposed when terminal is not a PTY